blues/application/tasks.py

```diff
@@ -224,24 +224,53 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository().get('url', '')
+    if url or not url.startswith('git@github.com:'):
```

Comment:
`url or` is redundant here, though it kinda makes sense but also makes harder to read so I would drop it.